tsconfig.build.json

```diff
@@ -0,0 +1,4 @@
+{
+  "extends": "./tsconfig",
+  "exclude": ["node_modules", "**/__tests__/*"]
```

Comment:
This was a useful pattern (from [this Stack Overflow answer](https://stackoverflow.com/a/58461810/732366))

Initially I was excluding the tests in `tsconfig.json` per the Typescript library tutorial I was following, but then the test files weren't picking up the Typescript config correctly

So `tsconfig.json` includes the tests for development purposes but then this extended `tsconfig.build.json` excludes them as far as what gets included in the published builds